packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-get-att.yaml

```diff
@@ -0,0 +1,32 @@
+Resources:
+  Bucket1:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: SomeBucketName
+      ReplicationConfiguration:
+        Role: "some-role"
+        Rules:
+        - Destination:
+            Bucket: Bucket1
+            StorageClass: STANDARD
+          Status: Enabled
```

Comment:
What's the point of all this? It doesn't use `GetAtt`, so I'm not sure what it contributes to this test.

Reply:
That's there so I can do `!GetAtt Bucket1.Rules.Destination` in the later buckets

Reply:
Actually ignore this for now, I've reworked this to be shorter

Reply:
But there is no return value `Rules.Destination` for `AWS::S3::Bucket`, right? https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html#aws-properties-s3-bucket-return-values-fn--getatt

Reply:
Indeed. This test is using invalid attributes, I've reworked this test to use different resources with valid attributes.